marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-developer.json

```diff
@@ -0,0 +1,12 @@
+{
+  "role-name": "hub-central-developer",
+  "description": "Permits developing and operating a Data Hub application via Hub Central",
+  "role": [
```

Comment:
seems to be missing:
hub-central-entity-model-writer and
hub-central-clear-user-data(should be allowed, since downloader is added?)

Reply:
Yes. I was missing the clear-user-data and explorer related roles from the list in the wiki. It should be resolved now.